packages/core/src/services/ai/providers/rules/vercel.test.ts

```diff
@@ -441,7 +441,7 @@ describe('applyVercelSdkRules', () => {
             type: 'tool-call',
             toolCallId: '123',
             toolName: 'toolName',
-            args: {
+            input: {
```

Comment:
They changed how they call the arguments of the function calls. Now they call it `input`